### PR TITLE
[2.12.x] - DDF-3924 Prevents more than one tab from showing at once

### PR DESCRIPTION
**Pull request by andrewkfiedler**
backport of https://github.com/codice/ddf/pull/3359

#### What does this PR do?
Prevents the possibility of more than one tab being shown at once in the even that multiple shown events get generated at the same time.

#### Who is reviewing it? 
@roelens8 
@mcalcote 
@mackncheesiest 
@djblue 
@clockard 

#### How should this be tested? (List steps with links to updated documentation)
Go to the admin UI and run either manually attempt to click tabs fast, or use this in the web console.
```$('[data-toggle=tab]').click()```
Verify only one tab gets shown at once.

#### What are the relevant tickets?
[DDF-3924](https://codice.atlassian.net/browse/DDF-3924)

### CODE REVIEW

File: ui/packages/ui/src/main/webapp/js/views/Module.view.js [hunk 1]
```diff
@@ -33,6 +33,29 @@ define([
...
+        events: {
+            'shown.bs.tab': 'tabShown'
+        },
+        tabShown: function(event){
+            if (this.$el.find('#tabs').find(event.target).length === 0) {
+                return;
+            }
+            var id = event.target.getAttribute('data-id');
+            this.tabs.currentView.children.each(function(childView) {
+                if (childView.model.id === id) {
```
Comment (mackncheesiest) [inline ui/packages/ui/src/main/webapp/js/views/Module.view.js]: ✏️ `childView.$el.toggleClass('active', childView.model.id === id)`

File: ui/packages/ui/src/main/webapp/js/views/Module.view.js [hunk 2]
```diff
@@ -33,6 +33,29 @@ define([
...
+                    childView.$el.addClass('active');
+                } else {
+                    childView.$el.removeClass('active');
+                }
+            });
+            this.tabContent.currentView.children.each(function(childView) {
+                if (childView.model.id === id) {
```
Comment (mackncheesiest) [inline ui/packages/ui/src/main/webapp/js/views/Module.view.js]: same ✏️

**Comment by andrewkfiedler** [comment]
build now

**Comment by vinamartin** [comment]
Looks like this is in 2.13.x with https://github.com/codice/ddf/pull/3359 
Should we close this out?

**Pull request closed without merging**